### PR TITLE
Fix: change 'js' to 'bash'

### PR DESCRIPTION
**Pull request by ghost**
1. The change is to change the sample from 'js' to 'bash'.
2. Some other typo changes.

### CODE REVIEW

**Review by XhmikosR — CHANGES_REQUESTED** [review]
`codes` is wrong IMO

**Review by XhmikosR — APPROVED** [review]

File: locale/en/knowledge/command-line/how-to-get-colors-on-the-command-line.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ When working on the command line, it can be both fun and extremely useful to col
 
 First, install it to the directory you'd like to work in.
 
-```js
+```bash
```
Comment (Trott) [inline locale/en/knowledge/command-line/how-to-get-colors-on-the-command-line.md]: We don't want bash highlighting here.

```suggestion
```console
```

File: locale/en/knowledge/command-line/how-to-get-colors-on-the-command-line.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ When working on the command line, it can be both fun and extremely useful to col
...
 cd mydir
```
Comment (Trott) [inline locale/en/knowledge/command-line/how-to-get-colors-on-the-command-line.md]: Remove this line, perhaps?

File: locale/en/knowledge/command-line/how-to-get-colors-on-the-command-line.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ console.log(colors.bold.grey.black.blue(stringThree));
 
 **Note**: Unlike the `String.prototype` approach, the chained methods on the `colors` instance are executed left to right i.e., the method closest to the string is finally applied. In the last `console.log` you can think of your terminal saying to itself, "Make this grey. Now, make this black. Now, make this blue. No more coloring methods now? Blue it is, then."
```
Comment (Trott) [inline locale/en/knowledge/command-line/how-to-get-colors-on-the-command-line.md]:
```suggestion
Unlike the `String.prototype` approach, the chained methods on the `colors` instance are executed left to right i.e., the method closest to the string is finally applied. In the last `console.log` you can think of your terminal saying to itself, "Make this grey. Now, make this black. Now, make this blue. No more coloring methods now? Blue it is, then."
```

**Comment by Trott** [comment]
This article should be removed in my opinion. It is inessential and outdated. 

* There's colorization support in Node.js itself now, and while it's a bit clunkier to use, it should be covered in an article like this.
* This focuses on an external module. But why choose `colors` and not `chalk` or another alternative?
* This is not something that needs to be covered in a knowledge guide. It is inessential. We don't need it, so let's remove it. We have too much stuff.

(Of course, in my opinion, all or nearly all of the knowledge guide should be removed.)

**Review by 3imed-jaberi — APPROVED** [review]

**Review by aymen94 — APPROVED** [review]

**Pull request merged**